### PR TITLE
fix(approval-signing-manager): hash large payloads

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Adds a fix for approval-signing manager, as seen in the vault and fireblocks signer

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: packages/approval-signing-manager/src/lib/approval-signing-manager.ts [hunk 1]
```diff
@@ -52,7 +52,9 @@ export class ApprovalSigner implements PolkadotSigner {
    * Get a signature from the approval key store. This may take a while as the approval process may require human approval
    */
   private async signData(address: string, data: Uint8Array): Promise<SignerResult> {
-    const message = `0x${Buffer.from(data).toString('hex')}`;
+    const fixedData = data.length > 256 ? blake2AsU8a(data) : data;
```
Comment (VictorVicente) [inline packages/approval-signing-manager/src/lib/approval-signing-manager.ts:55]: Is there any unit test for this line?

Comment (polymath-eric) [reply packages/approval-signing-manager/src/lib/approval-signing-manager.ts:55]: I added in tests to all signing managers.

As long as **this PR should be merged as two commits** and not squash and merge, the release shouldn't make unnecessary releases

**Review by polymath-eric — COMMENTED** [review]
This PR should be merged as two commits

**Comment by polymath-eric** [comment]
There is that merge commit in the middle of my two commits.

I am tempted to rebase it out since there is only one comment

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**